From b4e849cfa15e2fcc9f501858d93b16431a24c225 Mon Sep 17 00:00:00 2001
From: "shuai.li" <shuai.li@amlogic.com>
Date: Tue, 4 Nov 2014 14:49:47 +0800
Subject: [PATCH 5021/5965] PD#95725 audio: add uevent for usb sound card
 connect or remove; add a proc interface for user to retrieve the ID/card pair

fix compile error

Change-Id: I9a8bee4a6c3df2892749b5bd1f34d6dfef2eea6a
---
 sound/usb/card.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/usb/card.c b/sound/usb/card.c
index 019477607b6a..a95862479499 100755
--- a/sound/usb/card.c
+++ b/sound/usb/card.c
@@ -346,6 +346,7 @@ static int usb_audio_source_info_Insert(u32 usbid, int num)
     struct mutex usb_audio_mutex2;
     struct usb_audio_source_config u1;
     
+    memset(&u1, 0, sizeof(struct usb_audio_source_config));
     mutex_init(&usb_audio_mutex2);
     mutex_lock(&usb_audio_mutex2);
     for(i = 1; i <= num; i++)
-- 
2.19.0

